pkg/analyzer/nodes/node.go

```diff
@@ -101,7 +101,8 @@ func extractFilesFromDirectory(root string, matcher matcher.PrefixMatcher) (*fil
 		files: make(map[string]*fileMetadata),
 	}
 	m := metrics.FileExtractionMetrics{}
-	for _, dir := range []string{"etc/", "usr/share/rpm", "var/lib/rpm"} { // TODO(ROX-13771): Use range matcher.GetCommonPrefixDirs() again after fixing
+	// TODO(ROX-13771): Use `range matcher.GetCommonPrefixDirs()` again after fixing.
+	for _, dir := range []string{"etc/", "usr/share/rpm", "var/lib/rpm", "usr/share/buildinfo"} {
```

Comment:
Perhaps for another PR, but since we are now looking at the RHCOS-specific locations, do we still need to look at `etc/` or `var/lib/rpm` here? Those ultimately just symlink to the other two paths. If we do get rid of it, we should also remove the symlink file reading stuff we added, as it would no longer be needed (and could be potentially dangerous if not handled correctly. We already fixed a bug where it was possible for us to read our own file outside of the given `root`)